images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
My concern is the next line `git checkout v3.0.0-beta-3`. It's going to need some big warning that says you can't change it anymore without updating your repo.